### PR TITLE
Generate-code and big models update

### PR DESCRIPTION
**Pull request by GermanVor**

### CODE REVIEW

**Comment by GermanVor** [comment]
`yandex/cloud/cic` service breaks proto generation script (I did not include it to generate-code)

**Comment by GermanVor** [comment]
working example for assistants service

```
import { Session, cloudApi } from "@yandex-cloud/nodejs-sdk";

const getEnv = (envName: string, defaultValue?: string): string => {
    const envValue = process.env[envName] || defaultValue;

    if (!envValue) {
        throw new Error(`Env variable ${envName} is not defined`);
    }

    return envValue;
};

const iamToken = getEnv("YC_IAM_TOKEN");
const folderId = getEnv("YC_FOLDER_ID");

(async () => {
    const session = new Session({ iamToken });

    const assistantApi = cloudApi.ai.assistants_v1_assistant_service;
    const messageApi = cloudApi.ai.assistants_v1_threads_message_service;
    const threadApi = cloudApi.ai.assistants_v1_threads_thread_service;
    const runApi = cloudApi.ai.assistants_v1_runs_run_service;

    const assistantClient = session.client(assistantApi.AssistantServiceClient);
    const messageClient = session.client(messageApi.MessageServiceClient);
    const threadClient = session.client(threadApi.ThreadServiceClient);
    const runClient = session.client(runApi.RunServiceClient);

    const thread = await threadClient.create(
        threadApi.CreateThreadRequest.fromPartial({
            name: "qwerty",
            folderId,
        })
    );

    console.log({ thread });

    const assistant = await assistantClient.create(
        assistantApi.CreateAssistantRequest.fromPartial({
            name: "qwerty",
            folderId,
            modelUri: `gpt://${folderId}/yandexgpt/latest`,
        })
    );

    console.log({ assistant });
    
    const assistantId = assistant.id;
    const threadId = thread.id;

    const message = await messageClient.create(
        messageApi.CreateMessageRequest.fromPartial({
            threadId,
            content: {
                content: [{ text: { content: "qwerty" } }],
            },
        })
    );

    console.log({ message });

    const run = await runClient.create(
        runApi.CreateRunRequest.fromPartial({
            threadId,
            assistantId,
        })
    );

    console.log({ run });

    const a = runClient.listen(
        runApi.ListenRunRequest.fromPartial({ runId: run.id })
    );

    let lastStreamEvent: cloudApi.ai.assistants_v1_runs_run_service.StreamEvent | null =
        null;
    for await (const streamEvent of a) {
        lastStreamEvent = streamEvent;
    }

    console.dir({ lastStreamEvent });
    console.dir(lastStreamEvent.completedMessage.content.content);
})();
```

**Comment by GermanVor** [comment]
`The inferred type of this node exceeds the maximum length the compiler will serialize. An explicit type annotation is needed.` - I do not know how to solve it correctly (there were few files I fixed manually)

**Comment by nikolaymatrosov** [comment]
> `yandex/cloud/cic` service breaks proto generation script (I did not include it to generate-code)

Should be fixed in the latest `cloudapi` commit https://github.com/yandex-cloud/cloudapi/commit/ac83786bdd85effc92c822788ee14e5b6038460d

**Comment by nikolaymatrosov** [comment]
This PR introduces breaking changes due to the renaming of nearly all services, which affects backward compatibility. As such, these updates should be included in a new major release of the SDK.

However, the changes themselves may not be significant enough to justify a major release. It would be ideal for the new release to also address additional improvements, such as splitting the SDK code into packages and supporting different cloud regions.

**Comment by DavyJohnes** [comment]
> This PR introduces breaking changes due to the renaming of nearly all services, which affects backward compatibility. As such, these updates should be included in a new major release of the SDK.
> 
> However, the changes themselves may not be significant enough to justify a major release. It would be ideal for the new release to also address additional improvements, such as splitting the SDK code into packages and supporting different cloud regions.

Согласен. А давайте выпустим альфу нового мажора!

@GermanVor можешь, пожалуйста, изменить таргетную ветку для мержа на `alpha`?

**Comment by DavyJohnes** [comment]
@GermanVor в `feat` коммите нужно сделать `BREAKING CHANGE`, чтобы автоматика мажор выпустила.
https://www.conventionalcommits.org/en/v1.0.0/#summary

File: package.json [hunk 1]
```diff
@@ -66,9 +66,9 @@
     "typescript": "^4.5.4"
   },
   "scripts": {
-    "test": "cross-env NODE_OPTIONS=\"--max-old-space-size=4096\" jest -c config/jest.ts --passWithNoTests '.*\\.test\\.ts$'",
-    "lint": "eslint src config",
-    "build": "cross-env NODE_OPTIONS=\"--max-old-space-size=4096\" tsc -p .",
+    "test": "jest -c config/jest.ts --passWithNoTests '.*\\.test\\.ts$'",
+    "lint": "eslint src/ --quiet --fix",
```
Comment (DavyJohnes) [inline package.json:70]: Почему конфиги не надо линтить?

File: package.json [hunk 2]
```diff
@@ -66,9 +66,9 @@
     "typescript": "^4.5.4"
   },
   "scripts": {
-    "test": "cross-env NODE_OPTIONS=\"--max-old-space-size=4096\" jest -c config/jest.ts --passWithNoTests '.*\\.test\\.ts$'",
```
Comment (DavyJohnes) [inline package.json:69]: Почему увеличение памяти больше не требуется?

**Review by DavyJohnes — COMMENTED** [review]
А давай `v1` префикс\суффикс вообще игнорить? Добавлять `vX` только для `X` > 1? Сейчас сильно избыточно выглядит.

File: package.json (hunk 2 above)
Comment (GermanVor) [reply package.json]: [d562dda](https://github.com/yandex-cloud/nodejs-sdk/pull/168/commits/d562dda0d5fd91583d5380e769c4c7b3ec300411)

**Review by DavyJohnes — APPROVED** [review]

**Pull request merged**